CHANGELOG.md

```diff
@@ -14,9 +14,66 @@ twilio-video.js now allows you to create and join a [Large Room](TODO_doc_link),
 
 - A RemoteAudioTrack of a Participant that is not active in the Large Room will now be switched off. (VIDEO-8668)
 - RemoteAudioTrack and RemoteVideoTrack will now have an additional property called [`switchOffReason`](TODO_doc_link),
-  which describes the reason for it being switched off. (VIDEO-8670, VIDEO-8748)
+  which describes the reason for it being switched off. The `switchOff` event will also have this reason string as a second
+  argument. (VIDEO-8670, VIDEO-8748)
 - A RemoteVideoTrack will now be switched off if the number of switched on RemoteVideoTracks reached the limit set
   by the media server. (VIDEO-8745)
+- Renamed BandwidthProfileOptions to BandwidthProfile. (VIDEO-8754)
+- Renamed VideoBandwidthProfileOptions to VideoBandwidthProfile. (VIDEO-8753)
+- The `isEnabled` property of the RemoteTrack is deprecated and scheduled for removal. Alternatively, you can determine if
```

Comment:
For the deprecations mentioned in here, is there a reason why we should just not remove/update them?

Reply:
We typically don't remove them right away. There is a deprecation period after which we remove the properties.

Reply:
My thought was, deprecate features only on minor/patch updates while waiting on a major bump. But since this release is in a major update, it doesn't make sense to add new deprecations if the customer have to do migration anyway for other breaking changes.

Reply:
Let's keep it deprecated for the duration of the pilot. I will propose that we remove them for GA. We need team-wide agreement for this.

Reply:
Sounds good.